corehq/apps/cachehq/cachemodels.py

```diff
@@ -110,11 +110,3 @@ class UserReportsDataSourceCache(GenerationCache):
     views = [
         'userreports/data_sources_by_build_info',
     ]
-
-
-class UserReportsReportConfigCache(GenerationCache):
```

Comment:
(related fogbugz ticket: http://manage.dimagi.com/default.asp?188213 and "fix" https://github.com/dimagi/commcare-hq/pull/9344)